README.md

```diff
@@ -4,6 +4,17 @@
 
 This library is compatible with Rust **1.41.1** or higher.
 
+However, if you want to use the library with 1.41, you will need to pin a couple of
+our dependencies:
+
+```
+cargo update -p serde --precise 1.0.156
+cargo update -p syn --precise 1.0.107
+```
+
+before building. (And if your code is a library, your downstream users will need to
+run these commands, and so on.)
```

Comment:
Sure, we can definitely copy this over to rust-bitcoin, and to rust-secp.

I think rust-secp probably does have pinning in its CI, since it has a `serde` feature, but it might not be in the README.

Reply:
phew that was hard, `serde` only uses `syn` if the "derive" feature is enabled and we do not enable it in secp.

Reply:
Ah, right! derp.